### PR TITLE
type: add page

### PR DESCRIPTION
**Pull request by yshhuang**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/type.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# type
+
+> Display the kind of command the shell will execute.
+
+- Display the kind of a command:
+
+`type {{command}}`
+
+- Display all locations containing an executable named {{command}}:
```
Comment (pxgamer) [inline pages/linux/type.md]:
```suggestion
- Display all locations containing the specified executable:
```

File: pages/linux/type.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# type
+
+> Display the kind of command the shell will execute.
```
Comment (pxgamer) [inline pages/linux/type.md]: Maybe it's worth using `type` instead of `kind`?
```suggestion
> Display the type of command the shell will execute.
```

Comment (pxgamer) [reply pages/linux/type.md]: Not sure if this is really necessary.

**Comment by sbrl** [comment]
Hey, @yshhuang! It looks like we're ready to merge. Could you sign the CLA please? @CLAassistant has posted a link above that you can click to do so.

**Pull request merged**